src/amr/data/field/field_geometry.hpp

```diff
@@ -14,68 +17,76 @@
 #include <SAMRAI/hier/BoxGeometry.h>
 
 
-namespace PHARE::amr
+namespace PHARE
```

Comment:
`namespace PHARE::amr`

for less indentation?

Reply:
sure, I'm always a bit reluctant to do so because it makes a big diff for nothing

Reply:
in this case you're making a diff by adding indentation ;)